### ISSUE TITLE
[Feature Request] Ability to exclude certain routes from Routes tab

### ISSUE THREAD

**Issue by hailwood** [itsgoingd/clockwork#493]
Right now we can tell clockwork to only collect requests from certain routes or to collect from all routes except a few, but the routes tab still shows all routes.

It would be great if we had a config option `routes.except` or `routes.only` that worked the same.

Use case is that I don't really care about the clockwork routes, the horizon routes, the pretty-routes routes, the ignition routes, the ... you get the idea, I want to be able to filter the list down to just show me routes I'm likely to be interacting with.

**Comment by itsgoingd** [comment]
Hey, I'm not sure how common of a use case this is. Is the routes tab something you use often?

I guess if we would implement something like this it would be in the form of "hide third-party routes" option.

**Comment by hailwood** [comment]
Hey @itsgoingd,

It is, I prefer the output to that of route:list and since it's there use it over something like the pretty-routes package.
How do you see the hide third-party routes working? Is there some way to know the source of the route definition?

**Comment by itsgoingd** [comment]
I was thinking checking the controller namespace against the application namespace might be enough to detect most third-party routes and would be fairly simple to implement.

**Comment by hailwood** [comment]
Could we add an array of controller namespaces which defaults to just the app namespace?
That would make it easy to whitelist specific third party packages that you do want to see

**Comment by itsgoingd** [comment]
This is now implemented here - https://github.com/itsgoingd/clockwork/pull/504

I've ended up scrapping the idea of not collecting third-party routes as detecting the application namespace was way too awkward. Instead I've added an option to simply list an array of namespaces to collect routes from, which is closer to your original idea I guess.

I'm not yet sure if we want to collect all or only the App namespace by default.
